vaadin-login-flow/src/main/java/com/vaadin/flow/component/login/AbstractLogin.java

```diff
@@ -106,6 +106,25 @@ public boolean isError() {
         return getElement().getProperty("error", false);
     }
 
+    /**
+     * Sets whether to show or hide the forgot password button.
+     * The button is visible by default
+     *
+     * @see #isForgotPasswordButtonVisible()
+     */
+    public void setForgotPasswordVisible(boolean forgotPasswordVisible) {
+        getElement().setProperty("noForgotPassword", !forgotPasswordVisible);
```

Comment:
Non-blocking: I would consider extracting the `"noForgotPassword"` literal into a constant since it's also used elsewhere.